README.md

```diff
@@ -1,2 +1,52 @@
 # 2019-winter-Haskell-school
+
 2019 winter Haskell school in Sydney Australia 
+
+
+
+## Class timetables and planning
+
+This winter session is initially designed to have 10 classes(might extended).
```

Comment:
CIS194 is a 12 week course and we will extend it with advanced contents.

Reply:
updated.